.versup.json

```diff
@@ -0,0 +1,16 @@
+{
+    "files": {
+        "source/wiz/_version.py": [
+            ["__version__ = \"([\\d\\.]+)\"", "__version__ = \"[version]\""]
+        ],
+        "doc/release/release_notes.rst": [
+            [
+                ".. release:: Upcoming",
+                ".. release:: [version]\n    :date: [version_date]"
+            ]
+        ]
+    },
+    "changelog": {
+        "create": false
```

Comment:
It doesn't seem to work, the changelog is still being generated:

```
(dev-py27) mb647:wiz > versup patch -n
In file source/wiz/_version.py replace __version__ = "3.1.2" with __version__ = "3.1.3"
✓ Updated doc/release/release_notes.rst, source/wiz/_version.py

Writing changelog entries:

- Fix incorrect type in versup file
- Add versup to project for versioning


✓ Changelog updated

Create commit with commit msg: Update version to 3.1.3
✓ Commit created

Create tag 3.1.3 with msg v3.1.3
✓ Tag v3.1.3 created
```

Reply:
Fixed in recent commit 2686760